spec/models/instructor_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Instructor do
```

Comment:
Should test happy path here as well (successful creation of a valid instructor object)